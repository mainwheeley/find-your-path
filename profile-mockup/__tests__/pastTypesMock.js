import React from 'react';
import { View, Text, StyleSheet } from 'react-native';
import PastData from '../comps/components/pastData';
import { shallow } from 'enzyme';
describe('Profile Bar', function() {
    it('renders correctly', () => {
	const tree = shallow(<PastData />)
	expect(tree).toMatchSnapshot();
    });
    it('length 1', function() {
	const tree = shallow(<PastData />)
	expect(tree.length).toBe(1);
    });
});
