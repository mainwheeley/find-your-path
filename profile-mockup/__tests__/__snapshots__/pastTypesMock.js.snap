// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Bar renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <View
    style={
      Object {
        "flex": 1,
        "justifyContent": "center",
        "margin": 10,
      }
    }
  >
    <FlatList
      ItemSeparatorComponent={[Function]}
      data={
        Array [
          Object {
            "key": "1",
            "startp": "N. Martin Jischke Dr",
            "totmiles": 5,
            "type": "Walk",
          },
          Object {
            "key": "2",
            "startp": "610 Purdue Mall",
            "totmiles": 5,
            "type": "Run",
          },
          Object {
            "key": "3",
            "startp": "230 S Grant St.",
            "totmiles": 4,
            "type": "Walk",
          },
          Object {
            "key": "4",
            "startp": "134 W State St.",
            "totmiles": 6,
            "type": "Bike",
          },
        ]
      }
      disableVirtualization={false}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      numColumns={1}
      onEndReachedThreshold={2}
      renderItem={[Function]}
      scrollEventThrottle={50}
      updateCellsBatchingPeriod={50}
      windowSize={21}
    />
  </View>,
  "nodes": Array [
    <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "center",
          "margin": 10,
        }
      }
    >
      <FlatList
        ItemSeparatorComponent={[Function]}
        data={
          Array [
            Object {
              "key": "1",
              "startp": "N. Martin Jischke Dr",
              "totmiles": 5,
              "type": "Walk",
            },
            Object {
              "key": "2",
              "startp": "610 Purdue Mall",
              "totmiles": 5,
              "type": "Run",
            },
            Object {
              "key": "3",
              "startp": "230 S Grant St.",
              "totmiles": 4,
              "type": "Walk",
            },
            Object {
              "key": "4",
              "startp": "134 W State St.",
              "totmiles": 6,
              "type": "Bike",
            },
          ]
        }
        disableVirtualization={false}
        horizontal={false}
        initialNumToRender={10}
        keyExtractor={[Function]}
        maxToRenderPerBatch={10}
        numColumns={1}
        onEndReachedThreshold={2}
        renderItem={[Function]}
        scrollEventThrottle={50}
        updateCellsBatchingPeriod={50}
        windowSize={21}
      />
    </View>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <PastData />,
    "_instance": PastData {
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": Object {},
      "updater": Updater {
        "_renderer": [Circular],
      },
      "workoutSeparator": [Function],
    },
    "_newState": null,
    "_rendered": <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "center",
          "margin": 10,
        }
      }
    >
      <FlatList
        ItemSeparatorComponent={[Function]}
        data={
          Array [
            Object {
              "key": "1",
              "startp": "N. Martin Jischke Dr",
              "totmiles": 5,
              "type": "Walk",
            },
            Object {
              "key": "2",
              "startp": "610 Purdue Mall",
              "totmiles": 5,
              "type": "Run",
            },
            Object {
              "key": "3",
              "startp": "230 S Grant St.",
              "totmiles": 4,
              "type": "Walk",
            },
            Object {
              "key": "4",
              "startp": "134 W State St.",
              "totmiles": 6,
              "type": "Bike",
            },
          ]
        }
        disableVirtualization={false}
        horizontal={false}
        initialNumToRender={10}
        keyExtractor={[Function]}
        maxToRenderPerBatch={10}
        numColumns={1}
        onEndReachedThreshold={2}
        renderItem={[Function]}
        scrollEventThrottle={50}
        updateCellsBatchingPeriod={50}
        windowSize={21}
      />
    </View>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <PastData />,
}
`;
